deps/graph-parser/src/logseq/graph_parser/block.cljs

```diff
@@ -35,27 +35,31 @@
          (string/join))))
 
 (defn- get-page-reference
-  [block]
+  [block format]
   (let [page (cond
                (and (vector? block) (= "Link" (first block)))
-               (let [typ (first (:url (second block)))
+               (let [url-type (first (:url (second block)))
                      value (second (:url (second block)))]
                  ;; {:url ["File" "file:../pages/hello_world.org"], :label [["Plain" "hello world"]], :title nil}
                  (or
                   (and
-                   (= typ "Page_ref")
+                   (= url-type "Page_ref")
                    (and (string? value)
                         (not (or (gp-config/local-asset? value)
                                  (gp-config/draw? value))))
                    value)
 
                   (and
-                   (= typ "Search")
+                   (= url-type "Search")
                    (page-ref/page-ref? value)
                    (text/page-ref-un-brackets! value))
 
+                  (and (= url-type "Search")
```

Comment:
Looking back at the [previous implementation](https://github.com/logseq/logseq/pull/9353/files#r1190619859), protocol links aren't relevant since their url-type is Complex. I'm not sure if asset links matter as I was unable to get a working asset link in org mode

Reply:
> I was unable to get a working asset link in org mode

hi, to get working asset links (tested on 0.9.6):
on windows:
- if in `/assets`, both `[[../assets/test.txt][test asset]]` and `[[file://..\assets\test.txt][test asset]]` will work (note the differnt slash directions; a single slash after `file` prefix will not work)
- if outside `/assets`, both `[[file:/D:\\Downloads\test.txt][test asset]]` and `[[file://D:\\Downloads\test.txt][test asset]]` will work (`file:/` prefix is always required; a single slash also works.)

the equivalent for linux is also true, except the slash direction situation is simpler.
PS. an image file will always be inlined in logseq, even if you add a different text label (not complaining)

Reply:
@lef-est Thanks for the examples! I thought I had tried the first example but I must've messed it up somehow. Updated PR to handle asset links